doc/mboxgrep.1

```diff
@@ -84,6 +84,8 @@ Write matches to file, pipe to stdout the rest. Useful to split a file
 in parts (requires -o).
 .IP "-x, --show-progress"
 Print progress while running.
+.IP "-z, --crlf"
+Use Windows new line (input and output).
```

Comment:
For reasons already explained, I am not willing to merge this.

Reply:
Didn't we say that we'd make it an option rather than automatic? Not sure how to otherwise implement this feature. you can take it out of the pr if you like.

Reply:
After a further consideration, I decided I don't want this feature. It will create more problems than it solves.